modules/ObjectControl/src/objectcontrol.cpp

```diff
@@ -690,6 +714,91 @@ void ObjectControl::startScenario() {
 	}
 }
 
```

Comment:
Add documentation to all functions

Reply:
I added some documentation in the hpp-file, do you think it's enough? I noticed there is no documentation for the rest of the code.

Reply:
A lot of the old code doesn't have much documentation, but for the newer code we have documented the code in the `.cpp`-files. See for instance `OSIAdapter` or `EsminiAdapter`.

Reply:
Added some more documentation, the functions are pretty straight forward in my opinion.